src/loader.rs

```diff
@@ -240,4 +277,144 @@ impl<'b, 'w> Loader<'b, 'w> {
         self.scene.add_mesh(mesh_handle.clone());
         Ok(mesh_handle)
     }
+
+    async fn load_video_clip(
+        &mut self,
+        video_clip_obj: object::video::ClipHandle<'_>,
+    ) -> anyhow::Result<Image> {
+        debug!("Loading texture image: {:?}", video_clip_obj);
+
+        let relative_filename = video_clip_obj
+            .relative_filename()
+            .context("Failed to get relative filename of texture image")?;
+        trace!("Relative filename: {:?}", relative_filename);
+        let file_ext = Path::new(&relative_filename)
+            .extension()
+            .unwrap()
+            .to_str()
+            .unwrap()
+            .to_ascii_lowercase();
+        let image: Vec<u8> = if let Some(content) = video_clip_obj.content() {
+            // TODO: the clone here is absolutely unnecessary, but there
+            // is no way to reconciliate its lifetime with the other branch of
+            // this if/else
+            content.to_vec()
+        } else {
+            let parent = self.load_context.path().parent().unwrap();
+            let image_path = parent.join(relative_filename);
+            self.load_context.read_asset_bytes(image_path).await?
+        };
+        let is_srgb = false; // TODO
```

Comment:
There is `srgb` crate which we could use as replacement for `rgb` since bevy uses sRGB by default.